### DISCUSSION TITLE
Are there any examples of using Wix v.4 anywhere?

### DISCUSSION THREAD

**Discussion by Zerpico** [wixtoolset/issues#7256] [Questions]
### WiX Toolset Page URL

https://wixtoolset.org/docs/v4/wixui/wixui_customizations/

### Page URL that linked to WiX Toolset

https://wixtoolset.org/docs/v4/wixui/wixui_customizations/

### Additional information (optional)

I didn't find any example of how to use wix.v.4, except for the default template from Extension HeatWave for Visual Studio. It is especially interesting to create an installer with WixUI

**Comment by jmp75** [comment]
I am trying to set up an installer using WiX4, and like you was looking for examples to start form. The documentation is understandably patchy given how new WiX4 is: doc is not easy...
I am capturing my journey in [blog posts](https://jmp75.github.io/work-blog/posts/20230315-windows-installer-wix/) where I list some resources, if it helps. In particular I found the [weekly live stream deployment dojo](https://www.youtube.com/watch?v=-Y7dOtciOJw&list=PLDlzbQXIs18slmqmdlS10_de_Cps-QRg6) by Rob Mensching.  

I am hoping to have a working installer within a few weeks for my particular use case.

**Reply by robmen** [reply]
Fortunately, there are very few changes in UI in v4. So, much of what you read about v3 UI applies to v4 UI, except for those things called out in the migration doc like [Converting custom WixUI dialog sets](https://wixtoolset.org/docs/fourthree/faqs/#converting-custom-wixui-dialog-sets).
